internal/acceptance/metastore_test.go

```diff
@@ -88,3 +100,28 @@ func runMetastoreTest(t *testing.T, extraAttributes map[string]any) {
 		}`, template),
 	})
 }
+
+func runMetastoreTestWithOwnerUpdates(t *testing.T, extraAttributes map[string]any) {
+	template := getTemplateFromExtraAttributes(t, extraAttributes)
+	unityAccountLevel(t, step{
+		Template: fmt.Sprintf(`resource "databricks_metastore" "this" {
+			name = "{var.STICKY_RANDOM}"
+			force_destroy = true
+			%s
+		}`, template),
+	}, step{
+		Template: fmt.Sprintf(`resource "databricks_metastore" "this" {
+			name = "{var.STICKY_RANDOM}"
+			force_destroy = true
+			owner = "account users"
+			%s
+		}`, template),
+	}, step{
+		Template: fmt.Sprintf(`resource "databricks_metastore" "this" {
+			name = "{var.STICKY_RANDOM}"
+			force_destroy = true
+			owner = "{env.TEST_METASTORE_ADMIN_GROUP_NAME}"
+			%s
+		}`, template),
```

Comment:
Do you need to test owner update twice? IIUC: we should test 1) owner update by itself, 2) both owner & other param update at the same time, to be 100% sure we cover the main cases. (Other param update by itself would be covered by existing integration tests.)

Reply:
I added another param update by itself in between steps to be sure there isn't any behaviour change in between steps without non owner changes. I will remove it.